tracking:
    ros__parameters:

        map:
            size: 20.0
            name: "world"
            resolution: 0.05
        
        bezier:
            m: 3
            tau: 0.5

        tracker:
            fov: 135.0
            range: 6.0
            radius: 0.15
            max_vel: 1.7
            max_acc: 3.0
            max_omega: 1.2
            max_alpha: 2.0
        
        planner:
            kappa: 8
            samples: 3
            delta_t: 0.1
            epsilon: 0.25
            num_angles: 32
            observation_distance: 2.5

            lambda_a: 1e0
            lambda_d: 1e0
            lambda_o: 1e0
            lambda_c: 1e4
            lambda_f: 1e5
            lambda_s: 1e3
        
        lbfgs:
            past: 3
            eps: 1e-6
            memory: 16    # default: 8
            delta: 1e-6
            epsilon: 1e-6
            max_step: 1e20

            # For Lewis-Overton line search
            wolfe: 9e-1
            armijo: 1e-4
            iterations: 64